### PR TITLE
game interface t6 file bus mode support

### PR DESCRIPTION
**Pull request by INSANEMODE**
Let me know what you think of the info at the top of the script, or if you have ideas for a better name for the script.

### CODE REVIEW

File: IW4MAdmin.sln
Comment (RaidMax) [inline IW4MAdmin.sln:1]: Do you mind removing these changes? I think it's just the newer version of VS having fun.

The only change in the SLN file should be

```diff
GameFiles\GameInterface\example_module.gsc = GameFiles\GameInterface\example_module.gsc
+GameFiles\GameInterface\_integration_t6_file_bus.gsc = GameFiles\GameInterface\_integration_t6_file_bus.gsc

File: _integration_t6_file_bus.gsc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+SetInboundData(location, data )
+{
+	writeFile( location, data );
+}
+
+SetOutboundData(location, data )
+{
+	writeFile(location, data );
+}
```
Comment (RaidMax) [inline _integration_t6_file_bus.gsc]: Add new line at end of file for consistency

File: IW4MAdmin.sln
Comment (INSANEMODE) [reply IW4MAdmin.sln:1]: yep, it was vs having fun, this has been fixed.

File: _integration_t6_file_bus.gsc (hunk 1 above)
Comment (INSANEMODE) [reply _integration_t6_file_bus.gsc]: new line added to end of file.

**Pull request merged**